### ISSUE TITLE
new physical resource

### ISSUE THREAD

**Issue by toneplex** [tongueroo/ufo#108]
I've been deploying to my Fargate stack without any issues, but starting last week, I get the message, `Requested update requires the creation of a new physical resource; hence creating one.` and then 3 hours later the message `UPDATE FAILED. ECS did not stabilize.`

This happened with both v 4.6.2 and v5.0.0.  I have yet to find a solution, so I was wondering if you've ever run into this?

**Comment by toneplex** [comment]
I don't see anything in the docs about which awscli version to use. Can you please let me know what the requirements are?

**Comment by toneplex** [comment]
I've been looking at the CloudFormation templates and I've noticed that in the working stack, the template has a section

```
Ecs:
    Type: AWS::ECS::Service
    DependsOn: Listener
    Properties:
      Cluster: stage
      DesiredCount: !If
        - EcsDesiredCountIsBlank
        - !Ref AWS::NoValue
        - !Ref EcsDesiredCount
      TaskDefinition: !Ref EcsTaskDefinition
      LaunchType: FARGATE
```

but in the template that is failing, there is no mention of `LaunchType: FARGATE`

You can see from the attached screenshot, it is trying to launch ECS instances instead of FARGATE.
![stack](https://user-images.githubusercontent.com/595547/84545867-aa110b80-acc5-11ea-83fd-b556a174ab81.png)

I'm using aws cli 2.0.22

**Comment by tongueroo** [comment]
Bummer. Think this may be v5 breaking fargate. 

Recommend using https://github.com/tongueroo/ufo/tree/v4.6.3 until have time to at closer look. 

Happy to consider PRs. No sweat either way of course. Definitely want this fixed though. It just may be a while. Swamped at the moment.

**Comment by toneplex** [comment]
Just confirming that reverting back to 4.6.3 has solved the issue.

**Comment by tongueroo** [comment]
Fixed fargate support in #109. Released in v5.0.1
